downstream/modules/platform/proc-create-password-hashes.adoc

```diff
@@ -0,0 +1,38 @@
+[id="proc-create-password-hashes"]
+
+= Creating PostgreSQL password hashes
+
+.Procedure
+. On your automation controller node, run the following:
++
+[literal, options="nowrap" subs="+quotes,attributes"]
+----
```

Comment:
Should the code be preceded with a `$` prompt?

Reply:
I don't know. It doesn't look much like command line code. At least not Linux. I think it's Python.